api/simplestore/sessions.go

```diff
@@ -0,0 +1,73 @@
+package simplestore
+
+import (
+	"database/sql"
+	"time"
+
+	"github.com/18F/e-QIP-prototype/api"
+	"github.com/pkg/errors"
+)
+
+// CreateOrUpdateSession returns session key or error
+func (s SimpleStore) CreateOrUpdateSession(accountID int, sessionKey string, expirationDate time.Time) error {
+	createQuery := `INSERT INTO Sessions (session_key, account_id, expiration_date)
+		VALUES ($1, $2, $3)
+		ON CONFLICT (account_id) DO UPDATE
+		SET session_key = $1, expiration_date = $3`
+
+	_, createErr := s.db.Exec(createQuery, sessionKey, accountID, expirationDate)
+	if createErr != nil {
+		return errors.Wrap(createErr, "Failed to create or update session")
+	}
+
+	return createErr
```

Comment:
createErr _must_ be nil here, so I think it's clearer to just return nil. It's also more idiomatic, returning nil makes it clear that this is the green path.